packages/hooks-react/src/useContentProtection.ts

```diff
@@ -20,20 +20,28 @@ const useContentProtection = <T>(
   const genericEntitlementService = getModule(GenericEntitlementService);
   const jwpEntitlementService = getModule(JWPEntitlementService);
 
-  const { configId, signingConfig, contentProtection, jwp, urlSigning } = useConfigStore(({ config }) => ({
+  const { configId, signingConfig, contentProtection, jwp, urlSigning, isAcessBridgeEnabled } = useConfigStore(({ config, settings }) => ({
     configId: config.id,
     signingConfig: config.contentSigningService,
     contentProtection: config.contentProtection,
     jwp: config.integrations.jwp,
     urlSigning: isTruthyCustomParamValue(config?.custom?.urlSigning),
+    isAcessBridgeEnabled: !!settings?.apiAccessBridgeUrl,
   }));
+
   const host = signingConfig?.host;
   const drmPolicyId = contentProtection?.drm?.defaultPolicyId ?? signingConfig?.drmPolicyId;
   const signingEnabled = !!urlSigning || !!host || (!!drmPolicyId && !host);
 
   const { data: token, isLoading } = useQuery(
     ['token', type, id, params],
     async () => {
+      if (isAcessBridgeEnabled) {
+        // if access control is set up we return nothing
+        // the useProtectedMedia hook will take care of it
+        return;
+      }
+
```

Comment:
This is a bit confusing, and I think you can switch it around. What if instead, we don't enable this query when isAccessBridgeEnabled === true and DO enable the next one if isAccessBridgeEnabled === true